src/modules/endpoint/DataContainer.js

```diff
@@ -87,6 +87,11 @@ export interface DataContainer {
    */
   eventsAvailable (city: string, language: string): Promise<boolean>,
 
+  /**
+   * Returns whether the city content (e.g. events, categories, ...) have been loaded or not.
+   */
+  cityContentAvailable (city: string, language: string): Promise<boolean>,
```

Comment:
What happens for city/language pairs that have only been peeked?
Maybe add this to the comment above